src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,119 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+
+  box-sizing: border-box;
+
+  width: 1024px;
```

Comment:
you shouldn't hardcode catalog width
it should be calculated depending on the product card width and max amount of cards